whisper_server.py

```diff
@@ -352,10 +356,14 @@ def transcribe_audio(self, audio_path):
                     "Darkstar, Texaco, Arco, Shell, Axeman, Darknight, Warrior, Pointer, Eyeball, "
                     "Moonbeam, Whiplash, Finger, Pinpoint, Ferret, Shaba, Playboy, Hammer, Jaguar, "
                     "Deathstar, Anvil, Firefly, Mantis, Badger. Also expect usage of the phonetic "
-                    "alphabet (Alpha, Bravo, Charlie, etc.). "

```

Comment:
Removed brackets as not expected to be in text. Not expecting `etc.` and added `X-ray` instead.